triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -900,6 +1020,22 @@ private[lf] class Runner private (
                   "Trigger rule state updated",
                   "state" -> triggerUserState(state, trigger.defn.level),
                 )
+                if (trigger.defn.level == Trigger.Level.High) {
```

Comment:
As these metrics are derived from high-level trigger data structures, they are not collected for low level triggers